extern/sector-storage/ffiwrapper/sealer_cgo.go

```diff
@@ -15,6 +15,8 @@ import (
 	"os"
 	"runtime"
 
+	nr "github.com/filecoin-project/lotus/extern/storage-sealing/lib/nullreader"
```

Comment:
Nit: We're generally trying to keep imports ordered in groups like this:
```
std

libs

filecoin-project/not-lotus

filecoin-project/lotus
```

So this should be at the bottom (..ideally we'd convince some linter to do this for us)

Reply:
check out go-fil-markets for at least one approach, I noticed they had automatic dependency rearranging when making a PR there last week